README.rst

```diff
@@ -180,6 +180,36 @@ If you intend to run on Mac OS, there are several more steps that are required.
   See the explanation for this step `here <https://github.com/ansible/ansible/issues/31869#issuecomment-337769174>`_.
 
 
+Running quipucords server in gunicorn
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can run the server locally inside of gunicorn.  This can be a useful way to debug.
+
+1. Clone the repository::
+
+    git clone git@github.com:quipucords/quipucords.git
+
+2. Switch to quipucords directory::
+
+    cd quipucords
```

Comment:
After testing this manually, this line is a little misleading because I feel like I am just supposed to cd to the repo I just cloned, but I actually am going to `quipucords/quipucords`